roscala/src/main/scala/coop/rchain/rosette/Ob.scala

```diff
@@ -87,8 +88,7 @@ trait Ob extends Base with Cloneable {
   def runtimeError(msg: String, state: VMState): (RblError, VMState) =
     (DeadThread, state)
 
-  def setAddr(indirect: Boolean, level: Int, offset: Int, value: Ob): Ob =
-    setLex(indirect, level, offset, value)._2
+  def setAddr(indirect: Boolean, level: Int, offset: Int, value: Ob): State[Ob, StoreResult] = ???
```

Comment:
Yeah, I doubt that we need this method at all in our translation from Rosette to Roscala. I have to take another look for which I currently don't have time since we want to demo a certain sequence of opcodes soon (and that sequence depends on this PR).

Reply:
I think that `setAddr` method is equal to `setLex`, so we could leave it as:
```scala
def setAddr: (Boolean, Int, Int, Ob) => State[Ob, toreResult] = setLex
```

Reply:
@tymm maybe we should have a rule of thumb, no ??? in `dev` or `master`. If sth is unknown then it should not be there. Worth even writting wart for it if there is none yet.

Reply:
@DoSOfRedRiver Sure, but it seems like it is used to save pointers. Something we don't want. On the other side I haven't observed it yet in gdb and don't know enough about its use cases to dismiss it entirely.
@rabbitonweb I am not sure if this rule of thumb makes sense for Roscala. Writing down just the type signature without an implementation helps us to keep PRs small while still building up the correct structure of our programs. I would totally agree with you if Roscala would not be the translation of an existing project.